examples/src/main/java/io/github/trimax/examples/plane/ambient/handlers/PlaneNoLightApplicationStartupHandler.java

```diff
@@ -0,0 +1,30 @@
+package io.github.trimax.examples.plane.ambient.handlers;
+
+import io.github.trimax.venta.engine.core.VentaContext;
+import io.github.trimax.venta.engine.interfaces.VentaEngineStartupHandler;
+import io.github.trimax.venta.engine.model.view.MeshView;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.joml.Vector3f;
+import org.joml.Vector4f;
+
+@Slf4j
+@AllArgsConstructor
+public final class PlaneNoLightApplicationStartupHandler implements VentaEngineStartupHandler {
+
+    @Override
+    public void onStartup(final String[] args, final VentaContext context) {
+        final var scene = context.getSceneManager().getCurrent();
+
+        final MeshView mesh = context.getMeshManager().load("plane");
+        final var plane = context.getObjectManager().create("plane", mesh, context.getProgramManager().load("basic"));
+        plane.getMesh().setMaterial(context.getMaterialManager().load("stone"));;
+        scene.add(plane);
```

Comment:
What do you think about defining an object `plane` in the resources and just calling `context.getObjectManager().load()`